tests/acceptance/stepDefinitions/provisioningContext.js

```diff
@@ -156,10 +166,19 @@ function blockUser(userId) {
 
 Given('user {string} has been created with default attributes', async function(userId) {
   await deleteUser(userId)
-  await createDefaultUser(userId)
+  await createDefaultUser(userId, 'large')
```

Comment:
is `large` now default skeleton dir??

Reply:
this step will be deleted after all the steps are refactored. So, keeping large as default in this step, as it used to use before this setup.

Reply:
if so, please add a comment as a reminder

Reply:
https://github.com/owncloud/QA/issues/659#issuecomment-803762470
to-do list has been created in this issue already. I don't think any other comment is necessary 🤔 
`adjust all uses of old steps in web to say "small", "large", or "without" skeleton`